swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/services/RemittancesPurgeTaskConsumer.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.billable.usage.services;
+
+import static com.redhat.swatch.billable.usage.configuration.Channels.REMITTANCES_PURGE_TASK;
+
+import com.redhat.swatch.billable.usage.configuration.ApplicationConfiguration;
+import com.redhat.swatch.billable.usage.data.BillableUsageRemittanceRepository;
+import com.redhat.swatch.billable.usage.model.EnabledOrgsResponse;
+import io.smallrye.reactive.messaging.annotations.Blocking;
+import jakarta.enterprise.context.ApplicationScoped;
+import java.time.OffsetDateTime;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.candlepin.clock.ApplicationClock;
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+
+@Slf4j
+@ApplicationScoped
+@AllArgsConstructor
+public class RemittancesPurgeTaskConsumer {
+  private final ApplicationClock applicationClock;
+  private final ApplicationConfiguration configuration;
+  private final BillableUsageRemittanceRepository remittanceRepository;
+
+  @Blocking
+  @Incoming(REMITTANCES_PURGE_TASK)
+  public void consume(EnabledOrgsResponse message) {
+    log.info("Received task for remittances purge with org ID: {}", message.getOrgId());
+    OffsetDateTime cutoffDate = getCutoffDate();
+    if (cutoffDate == null) {
+      return;
```

Comment:
Add a warning log here to indicate the the cutoff date was not configured and that nothing will be purged.

Reply:
done